File: 프로그래머스/2/42584. 주식가격/주식가격.py
def solution(prices):
    answer = []
    num = len(prices)
    
    for i in range(num):
        cnt = 0
        for j in range(i+1, num):
            cnt += 1
            if prices[i] > prices[j]:
                answer.append(j-i)
                break
        else:
            answer.append(num-1-i)
                
    return answer

File: 프로그래머스/2/42584. 주식가격/주식가격.js
function solution(prices) {
    const answer = [];
    
    for(let i = 0; i < prices.length; i++){
        const current = prices[i];
        let count = 0;
        
        for(let j = i+1; j < prices.length; j++){
            count++;
            
            if(prices[j] < current) break;

        }
        answer.push(count)
    }
    return answer
}